tools/Dashboard/DevHome.Dashboard/Views/DashboardView.xaml.cs

```diff
@@ -656,10 +666,7 @@ protected override void OnNavigatingFrom(NavigatingCancelEventArgs e)
         Log.Logger()?.ReportDebug("DashboardView", $"Leaving Dashboard, deactivating widgets.");
 
         // Deactivate widgets if we're not on the Dashboard.
-        foreach (var widget in PinnedWidgets)
-        {
-            widget.UnsubscribeFromWidgetUpdates();
-        }
+        UnsubscribeFromWidgets();

```

Comment:
Do we still need this here? Would `Unloaded` handle this case as well?

Reply:
It was not needed, but didn't hurt that much to fave. Removed.